policy-controller/core/src/http_route.rs

```diff
@@ -101,8 +101,8 @@ impl PartialOrd for GroupKindName {
     }
 }
 
-impl PartialEq for GroupKindName {
-    fn eq(&self, other: &Self) -> bool {
+impl GroupKindName {
+    pub fn eq_ignore_ascii_case(&self, other: &Self) -> bool {
```

Comment:
what's the motivation for moving this to a separate function rather than just making it the `PartialEq` behavior like we did before?

Reply:
If we manually implement PartialEq then we would need to also create a Hash impl which ignores ascii case.  To avoid this, I instead just created a separate function for when we want to do case insensitive comparisons.